honeycomb-core/src/cells/collections.rs

```diff
@@ -0,0 +1,43 @@
+//! Module short description
+//!
+//! Should you interact with this module directly?
+//!
+//! Content description if needed
+
+// ------ IMPORTS
+
+use crate::{CMap2, CoordsFloat, EdgeIdentifier, FaceIdentifier, VertexIdentifier};
+
+// ------ CONTENT
+
+pub struct VertexCollection<'a, T: CoordsFloat> {
+    map: std::marker::PhantomData<&'a CMap2<T>>,
+    pub identifiers: Vec<VertexIdentifier>,
```

Comment:
`map` is not a good name.

Reply:
renamed it to `lifetime_indicator` in decb9ac